html/functions.php

```diff
@@ -90,3 +90,34 @@ function createTweetreply($text, $user_id, $reply_id)
     return $stmt->execute();
 }
 /* 返信課題はここからのコードを修正しましょう。 */
+
+function createfavorite($member_id, $post_id)
```

Comment:
```suggestion
function createFavorite($member_id, $post_id)
```

プログラミングにおいて、単語と単語のつなぎ目が分かるようにすることが一般的です
（1つの単語の場合は、全部小文字を使えばOK）

主には
- キャメルケース（createFavorite）
- スネークケース（create_favorite）
の2種類があります（他の記述方法もありますが、↑↑↑の2つが一般的）

最低限1つのプロジェクト（= 作っている複数のコード郡）では統一しましょう。

以下、メソッド名は同じ指摘があるので全体を見直してください。